Assets/Mirror/Tests/Editor/NetworkIdentityTests.cs

```diff
@@ -284,14 +289,15 @@ public void OnStartServerTest()
             MyTestComponent component1 = gameObject.AddComponent<MyTestComponent>();
             MyTestComponent component2 = gameObject.AddComponent<MyTestComponent>();
 
-            identity.OnStartServer();
+            identity.OnStartServer.AddListener(component1.OnStartServer);
+            identity.OnStartServer.AddListener(component2.OnStartServer);
+
+            identity.OnStartServer.Invoke();
```

Comment:
you are just calling the event directly,  not much of a test.   Should this be identity.StartServer instead?

Reply:
This looks like I failed to update the test properly. I will add a commit for that and we can re-review.

Reply:
Resolved in recent commits. Please re-review.